.github/workflows/static_analysis.yml

```diff
@@ -0,0 +1,33 @@
+name: Static analysis
+on:
+  pull_request:
+
+jobs:
+  static_analysis:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
Could you try upgrading to `actions/checkout@v4`? (see: https://github.com/actions/checkout)

Reply:
I have upgraded both `actions/checkout@v4` and `subosito/flutter-action@v2` to the latest version instead of previous versions